config/environments/development.rb

```diff
@@ -48,8 +48,6 @@
   # config.action_view.raise_on_missing_translations = true
   ConfigHelper.setup_action_mailer(config)
 
-  config.rails_semantic_logger.semantic   = false
-  config.rails_semantic_logger.started    = true
-  config.rails_semantic_logger.processing = true
-  config.rails_semantic_logger.rendered   = true
+  # Change this to :json to simulate production
```

Comment:
is this just while you're working or is it meant as a permanent change?

Reply:
I out the comment there so someone knows that it's easy to toggle if they want. For some reason ENV var didn't work even with (ENV['LOGGER FORMAT'] || 'color).to_sym